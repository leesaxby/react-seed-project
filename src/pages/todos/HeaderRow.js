import React from 'react';
import { Row, Col, Image } from 'react-bootstrap';

import listImage from 'Images/list.png';

export default function HeaderRow() {
	return (
	    <Row>
	        <Col sm={1} lg={1} smOffset={4} lgOffset={5}>
	            <Image src={ listImage }
	                   alt="List Image"
	                   style={{ 'height': '200px',
	                            'marginBottom': '20px' }}/>
								asdasdsdasd
	        </Col>
	    </Row>
	);
}
