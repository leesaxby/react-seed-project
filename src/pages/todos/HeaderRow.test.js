import React from 'react';
import renderer from 'react-test-renderer';
import HeaderRow from './HeaderRow';

describe('A HeaderRow', () => {

	it('renders component', () => {
		const tree = renderer.create(
			<HeaderRow />
		).toJSON();

		expect(tree).toMatchSnapshot();
	});

});