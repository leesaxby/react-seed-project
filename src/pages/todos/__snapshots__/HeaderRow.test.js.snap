// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A HeaderRow renders component 1`] = `
<div
  className="MuiGrid-typeContainer-1 MuiGrid-spacing-xs-16-23 MuiGrid-justify-xs-center-18"
>
  <div
    className="MuiGrid-typeItem-2"
  >
    <img
      alt="header"
      src="test-file-stub"
      style={
        Object {
          "height": "200px",
          "marginBottom": "20px",
        }
      }
    />
  </div>
</div>
`;
